tests/SprykerSdkTest/Zed/AppSdk/Business/AppAsyncApiFacadeTest.php

```diff
@@ -193,4 +193,28 @@ public function testAddAsyncApiMessageFromTransferObjectWithSpecifiedMessageName
         $this->tester->assertAsyncApiResponseHasNoErrors($asyncApiResponseTransfer);
         $this->tester->assertAsyncApiHasPublishMessageInChannel($asyncApiRequestTransfer->getTargetFile(), 'FooBar', $asyncApiMessageTransfer->getChannel()->getName());
     }
+
+    /**
+     * @return void
+     */
+    public function testAddAsyncApiMessagePropertyAdd(): void
```

Comment:
```suggestion
    public function testAddAsyncApiMessageAddsPropertyToMessage(): void
```
The method name should be readable as it would be a normal sentence.

Reply:
I have changed the method name as suggested.